app/models/vacancy_publish_feedback.rb

```diff
@@ -17,7 +16,7 @@ def to_row
       user&.oid,
       vacancy.id,
       vacancy.school.urn,
-      rating,
+      nil, # we used to send rating, for now we just stop sending it
```

Comment:
Might be worth keeping this line, so that if we re-exported from blank then where ratings did exist, they would still get output.